fs/flail.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+# Copyright: 2018 IBM
+# Author: Praveen K Pandey <praveen@linux.vnet.ibm.com>
+#
+
+import os
+import re
+
+from avocado import Test
+from avocado import main
+from avocado.utils import process, build, archive
+from avocado.utils.software_manager import SoftwareManager
+
+
+class Flail(Test):
+    '''
+    Flail is system call fuzzer
+
+    :avocado: tags=fs
+    '''
+
+    def setUp(self):
+        '''
+        Setup Flail
+        '''
+        smm = SoftwareManager()
+        for package in ['gcc', 'make']:
+            if not smm.check_installed(package) and not smm.install(package):
+                self.cancel(package + ' is needed for the test to be run')
+
+        tarball = os.path.join(self.datadir, "flail-0.2.0.tar.gz")
```

Comment:
Is flail not maintained in upstream ?

Reply:
@narasimhan-v no we got this test from internal

Reply:
@PraveenPenguin my suggestion would be to have it in upstream (maybe create a git if owner is fine). If that does not work out, then we can have tarball here.

Reply:
@PraveenPenguin We can contact the maintainer, and if no response, then we can push this commit.
How long can we wait for the maintainer's response ? any suggestions ?

Reply:
@narasimhan-v i think we can have it here only as i did not get any response from maintainer , will send V2 as soon as possible